Packs/AccentureCTI_Feed/Integrations/ACTIIndicatorFeed/ACTIIndicatorFeed.py

```diff
@@ -0,0 +1,146 @@
+                      ' is correct and that you have access to the server from your host.' \
+                      '\nError Type: {}\nError Number: [{}]\nMessage: {}\n' \
+                .format(err_type, exception.errno, exception.strerror)
+            raise DemistoException(err_msg, exception)                                                        # pragma: no cover
+
+    demisto.debug(f"Received in total {len(result)} indicators from ACTI Feed")
+    return result
+
+
+def create_fetch_configuration(indicators_type: list, filters: dict, params: dict) -> Dict[str, dict]:
+    mapping_by_indicator_type = {                                                                             # pragma: no cover
+        'IP': {
+            'last_seen_as': 'malwaretypes',
+            'threat_types': 'primarymotivation',
+            'malware_family': 'malwarefamily',
+            'severity': 'sourceoriginalseverity'},
+        'Domain': {
+            'last_seen_as': 'malwaretypes',
+            'threat_types': 'primarymotivation',
+            'malware_family': 'malwarefamily',
+            'severity': 'sourceoriginalseverity'},
+        'URL': {
+            'last_seen_as': 'malwaretypes',
+            'threat_types': 'primarymotivation',
+            'malware_family': 'malwarefamily',
+            'severity': 'sourceoriginalseverity'}
+    }
+
+    url_by_type = {"IP": 'https://api.intelgraph.idefense.com/rest/threatindicator/v0/ip',                    # pragma: no cover
+                   "Domain": 'https://api.intelgraph.idefense.com/rest/threatindicator/v0/domain',
+                   "URL": 'https://api.intelgraph.idefense.com/rest/threatindicator/v0/url'}
+
+    common_conf = {'extractor': 'results',
+                   'indicator': 'display_text',
+                   'insecure': params.get('insecure', False),
+                   'custom_build_iterator': custom_build_iterator,
+                   'filters': filters}
+
+    indicators_configuration = {}
+
+    for ind in indicators_type:
+        indicators_configuration[ind] = dict(common_conf)
+        indicators_configuration[ind].update({'url': url_by_type[ind]})
+        indicators_configuration[ind].update({'indicator_type': ind})
+        indicators_configuration[ind].update({'mapping': mapping_by_indicator_type[ind]})
+
+    return indicators_configuration
+
+
+def build_feed_filters(params: dict) -> Dict[str, Optional[Union[str, list]]]:
+    filters = {'severity.from': params.get('severity'),
+               'threat_types.values': params.get('threat_type'),
+               'confidence.from': params.get('confidence_from'),
+               'malware_family.values': params.get('malware_family', '').split(',')
+               if params.get('malware_family') else None}
+
+    return {k: v for k, v in filters.items() if v is not None}
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+    parameters = demisto.params()
```

Comment:
No need for double instantiation of parameters. You can remove `parameters` and stick with just `params`.

Reply:
I have tested before pushing the changes for this, but shows a mypy error:
https://app.circleci.com/pipelines/github/demisto/content/139152/workflows/d1eecee1-daa9-4c3c-ac37-2843bfe2c8d1/jobs/471032?invite=true#step-107-53

Reply:
@satyakidroid - I believe this is because `params` is returned as `None` instead of `dict`. You need to debug why `params` is coming back undefined.

Reply:
It's saying : `Integration log: <class 'dict'>`

Reply:
@satyakidroid - Can you try to change `params = demisto.params()` instead of ~~`params = {k: v for k, v in demisto.params().items() if v is not None}`~~ in line 133 and see if it's still an issue?

Reply:
fixed here [`a975606` (#17099)](https://github.com/demisto/content/pull/17099/commits/a975606865cd5255ab3912fbbc99d2320ae7de3c).